server/test/services/export/PdfExporterTest.java

```diff
@@ -160,31 +284,39 @@ public void exportApplication_optionalFileUploadWithFile_asAdmin()
   @Test
   public void exportApplication_optionalFileUploadWithoutFile()
       throws IOException, DocumentException {
-    createFakeProgramWithOptionalQuestion();
+
```

Comment:
Blocking - @devin-civi  - Is there a test which tests both optional and required file? Example - income document may be required document for a program but property doc may be optional document. Can we add it to the test?

Reply:
There isn't, but there wasn't in the original tests either, and it doesn't add that much coverage as far as I can tell. There are other tests that handle required questions in general, it doesn't seem like we need a test specifically for file upload.

Note the only reason "optional" matters for these tests is because we want to test a file upload with no answer.